qase-javascript-commons/src/reporters/abstract-reporter.ts

```diff
@@ -68,6 +62,32 @@ export abstract class AbstractReporter implements ReporterInterface {
     return this.results;
   }
 
+  /**
+   * @param {TestResultType} result
+   */
+  public addTestResult(result: TestResultType) {
+    if (result.testops_id === null || !Array.isArray(result.testops_id)) {
+      this.results.push(result);
+      return;
+    }
+
+    // if we have multiple ids, we need to create multiple test results and set duration to 0 for all but the first one
+    let firstCase = true;
+
+    for (const id of result.testops_id) {
+      const testResultCopy = { ...result };
+      testResultCopy.testops_id = id;
+      testResultCopy.id = uuidv4();
+
+      if (!firstCase) {
+        testResultCopy.execution.duration = 0;
+      }
+
+      firstCase = false;
+      this.results.push(result);
```

Comment:
shouldn't it be
```suggestion
      this.results.push(testResultCopy);
```
?

Reply:
fixed